_maps/lighthouseisland/lighthouseisland.yaml

```diff
@@ -0,0 +1,189 @@
+---
+name:
+  en: Lighthouse Island
+desc:
+  en: Job posting - Lightkeeper needed! Want to get paid to basically keep the lights on, all while enjoying endless ocean views on your own private island? Then this job might be for you. Responsibilites include helping ships & their crew navigate their cargo safely into the harbor at night. Apply today! 
+ruleSet: Standard
+theme: Mario
+initialCash: 2400
+targetAmount: 20000
+baseSalary: 360
+salaryIncrement: 225
+maxDiceRoll: 6
+frbFile1: lighthouse
+frbFile2: lighthouse2
+frbFile3: lighthouse3
+frbFile4: lighthouse4
+background: bg102alt
+mapIcon: lightICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/lighthouseisland.music.zip
+  - https://drive.google.com/file/d/1yoEBKlOkr9pJNWev-xWZ8nuIWdHYGhU2/view?usp=sharing

```

Comment:
This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1yoEBKlOkr9pJNWev-xWZ8nuIWdHYGhU2`